src/index.ts

```diff
@@ -3,4 +3,14 @@ import http from 'http';
 
 const server: Server = new Server();
 
-const app: http.Server = server.start();
+let app: http.Server;
+
+server
+  .start()
+  .then((application: http.Server) => {
+    app = application;
+  })
+  .catch(err => {
+    // tslint:disable-next-line:no-console
+    console.error(err);
```

Comment:
To avoid using console and disabling the TSLint rule, I included Signale way at the beginning. Import it in the file, and then you can do
```suggestion
    signale.fatal(err);
```

Reply:
Good, I would definitely rather do this.

Reply:
Made the change for you.